dev/io.openliberty.org.apache.myfaces.4.0_fat/test-applications/ClientWindowScopedTest.war/resources/clientFlow/page1.xhtml

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<!--
+    Copyright (c) 2023 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License 2.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-2.0/
+    
+    SPDX-License-Identifier: EPL-2.0
+ -->
+<html xmlns="http://www.w3.org/1999/xhtml" 
+	xmlns:f="jakarta.faces.core"
+	xmlns:h="jakarta.faces.html" 
+	xmlns:c="jakarta.tags.core">
+
+<h:head>
+	<title>Enter flow page 1</title>
+</h:head>
+
+<h:body>
+    <h:form id="form">
+        <h:commandLink id="check" value="check" action="#{flowBean.checkClient()}" />
+        <h:commandLink id="next" value="next" action="page2" />
+    </h:form>
+</h:body>
+
+</html>
```

Comment:
Page should be renamed to `clientFlow.xhtml` since flow is implicit. Otherwise, link the start node to page1.xhtml in the flow xml file.

Reply:
That should solve the exception.